src/api-reference/Notifications/PartnerNotifications.markdown

```diff
@@ -0,0 +1,165 @@
+---
+title: Notification Platform - Partner Notifications API
+layout: reference
+---
+{% include prerelease.html %}
+
+The purpose of this API is to provide SAP Concur's partners the ability to message users, through the web and mobile product.
+
+>**Sample use case:** A business traveller starts a travel booking. That user is notified that a visa is required for their trip.
+
+* [Process Flow](#process-flow)
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Dependencies](#dependencies)
+* [Access Token Usage](#access-token-usage)
+* [Send a message](#send-message)
+* [Schema](#schema)
+* [Request](#notification)
+* [Error](#schema-error)
+
+## <a name="process-flow"></a>Process Flow
+
+![ey](./ey.png)
+
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Travel Professional Edition
+* Concur Travel Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+Name|Description|
+---|---
+`notifications.messages.writeonly`|Write messages to the notifications platform|
+
+## <a name="dependencies"></a>Dependencies
```

Comment:
Can we add a dependency here that the domains to be passed must first be whitelisted?

Reply:
This is not a dependency for the API but i added a note within the request description. Please contact Concur to whitelist the domains of the URLs.

Reply:
The term 'whitelist' really should be on our no-fly zone.

https://twitter.com/dhh/status/1032050325513940992
https://twitter.com/andrestaltz/status/1030200563802230786